src/app/modules/ix-table2/components/ix-table-body/ix-table-body.component.spec.ts

```diff
@@ -1,22 +1,36 @@
+import { HarnessLoader, parallel } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { MatSlideToggleHarness } from '@angular/material/slide-toggle/testing';
 import { Spectator } from '@ngneat/spectator';
 import { createComponentFactory } from '@ngneat/spectator/jest';
 import { ArrayDataProvider } from 'app/modules/ix-table2/array-data-provider';
 import { textColumn } from 'app/modules/ix-table2/components/ix-table-body/cells/ix-cell-text/ix-cell-text.component';
+import { toggleColumn } from 'app/modules/ix-table2/components/ix-table-body/cells/ix-cell-toggle/ix-cell-toggle.component';
+import { yesNoColumn } from 'app/modules/ix-table2/components/ix-table-body/cells/ix-cell-yesno/ix-cell-yesno.component';
 import { IxTableBodyComponent } from 'app/modules/ix-table2/components/ix-table-body/ix-table-body.component';
 import { IxTable2Module } from 'app/modules/ix-table2/ix-table2.module';
 import { createTable } from 'app/modules/ix-table2/utils';
 
 interface TestTableData {
   numberField: number;
   stringField: string;
-  booleanField: boolean;
+  yesNoField: boolean;
+  enabledField: boolean;
 }
 
 const testTableData: TestTableData[] = [
-  { numberField: 1, stringField: 'a', booleanField: true },
-  { numberField: 2, stringField: 'c', booleanField: false },
-  { numberField: 4, stringField: 'b', booleanField: false },
-  { numberField: 3, stringField: 'd', booleanField: true },
+  {
```

Comment:
While having this kind of integration test is good, I think it would be better to (also) have more isolated tests in column specific folders.